diagnostic.md

```diff
@@ -20,6 +25,8 @@ c = c + d;
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
 
+c = 7
+'+' concatenates the variables on either side of it and that result is stored in c.
```

Comment:
"concatenate" is a term we use to describe the combining of strings or characters. In this instance, we are doing a simple add. What about the `=`? What does that do?